src/main/scad/yavuz-demirhan/basket-burr/basketBurr.scad

```diff
@@ -0,0 +1,57 @@
+include <puzzlecad.scad>
+
+// Global settings
+$burr_scale = 10;
+$plate_width = 100;
+$burr_inset = 0.05;
+$joint_inset = 0.08;
+$burr_bevel = 1.0;
+$unit_beveled = false;
+
+*basket();
+*holder();
+*pieces();
+
+module basket() {
+    render(convexity = 2)
+    burr_piece([
+        "xxxxxx|xxxxxx|x....x|......|......|......",
+        "xxxxxx|xxxx{connect=fy+}xx|x....x|......|......|......",
+        "xxxxxx|......|......|......|......|......",
+        "xxxxxx|......|......|......|......|......",
+        "xxxxxx|xxx{connect=fy+}xxx|x....x|......|......|......",
```

Comment:
I generally prefer oriented & labeled connectors - any specific reason for choosing square/unlabeled ones here?